dataset/AMJMVSDAVEAAY/build.json

```diff
@@ -0,0 +1,80 @@
+{
+  "id": "AMJMVSDAVEAAY",
+  "submitTime": "2021-09-03T02:52:04.997Z",
+  "startTime": "2021-09-03T02:52:05.416Z",
+  "endTime": "2021-09-03T02:57:10.284Z",
+  "progress": "FINISHED",
+  "status": "SUCCESS",
+  "buildContentId": "build-AMJMVSDAVEAAY",
+  "temporaryBuild": false,
+  "scmUrl": "http://code.stage.engineering.redhat.com/gerrit/opentracing-contrib/java-web-servlet-filter.git",
```

Comment:
Same problem as internal url in public repo.
If this json is some data file will be needed in real test, consider to store it in configmap or something similar in runtime env.